src/app/config/apis.ts

```diff
@@ -0,0 +1,299 @@
+        select: (data: any) =>
+          data.reduce(
+            (acc: number, curr: any) => curr.user_symbiotic_points + acc,
+            0
+          ),
+      },
+      {
+        getURL: (wallet: string) =>
+          `https://points.mellow.finance/v1/chain/1/defi/users/${wallet}`,
+        select: (data: any) =>
+          data.reduce(
+            (acc: number, curr: any) => curr.user_symbiotic_points + acc,
+            0
+          ),
+      },
+      {
+        getURL: (wallet: string) =>
+          `https://app.ether.fi/api/portfolio/v3/${wallet}`,
+        select: (data: any) =>
+          data.totalPointsSummaries?.SYMBIOTIC?.CurrentPoints || 0,
+      },
+    ],
+  },
+  {
+    pointsId: POINTS_ID_LOMBARD_LUX_S1,
+    dataSources: [
+      {
+        getURL: (wallet: string) =>
+          `https://mainnet.prod.lombard.finance/api/v1/referral-system/season-1/points/${wallet}`,
+        select: (data: any) => data?.total || 0,
+      },
+    ],
+  },
+  {
+    pointsId: POINTS_ID_RESOLV_S1,
+    dataSources: [
+      {
+        getURL: (wallet) =>
+          `https://api.resolv.xyz/points/leaderboard/slice?address=${wallet}`,
+        select: (data, wallet) => {
+          const rows = data?.rows || [];
+          for (const row of rows) {
+            if (row.address.toLowerCase() === wallet.toLowerCase()) {
+              return Number(row.points);
+            }
+          }
+          return 0;
+        },
+      },
+    ],
+  },
+  {
+    pointsId: POINTS_ID_MERITS_S1,
+    dataSources: [
+      {
+        getURL: (wallet: string) =>
+          `https://points.mellow.finance/v1/chain/1/users/${wallet}`,
+        select: (data: any) =>
+          data.reduce(
+            (acc: number, curr: { user_merits_points: number }) =>
+              curr.user_merits_points + acc,
+            0
+          ),
+      },
+    ],
+  },
+  {
+    pointsId: POINTS_ID_HYPERBEAT_S1,
+    dataSources: [
+      {
+        getURL: (wallet: string) => `https://app.hyperbeat.org/api/points`,
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        getBody: (
+          wallet: string,
+          startTimestamp?: number,
+          endTimestamp?: number
+        ) => {
+          const query = {
+            query:
+              formatGraphQLQuery(`query GetWeeklyPoints($user_filter: String!, $start_timestamp: bigint!, $end_timestamp: bigint!) {
+              cron_get_ranked_points_by_timestamp(args: {
+                provider_filter: "hyperbeat",
+                user_filter: $user_filter,
+                start_timestamp: $start_timestamp,
+                end_timestamp: $end_timestamp
+              }) {
+                total_value
+              }
+            }`),
+            variables: {
+              user_filter: wallet,
+              start_timestamp: 1743408000, // UI start timestamp, presumably start of all point accruals
+              end_timestamp: endTimestamp,
```

Comment:
Using current timestamp, but this is different than the Hyperbeat UI. As discussed, UI is using graphQL to mimic epochs. Wondering how we get the current endTimestamp the UI is using.